_pages/documentation/general_docs/gpu_models/amdgpu.md

```diff
@@ -1,44 +1,44 @@
 ---
 layout: documentation
-title: GCN3 GPU model
+title: AMD GPU model
 doc: gem5 documentation
 parent: gpu_models
-permalink: /documentation/general_docs/gpu_models/GCN3
+permalink: /documentation/general_docs/gpu_models/vega
 ---
 
-# **GCN3**
+# **AMD GPU model**
 
 Table of Contents
 
 1. [Using the model](#Using-the-model)
 2. [ROCm](#ROCm)
 3. [Documentation and Tutorials](#Documentation-and-Tutorials)
 
-The GCN3 GPU is a model that simulates a GPU at the ISA level, as opposed to the intermediate language level. This page will give you a general overview of how to use this model, the software stack the model uses, and provide resources that detail the model and how it is implemented.
+The AMD GPU is a model that simulates a GPU at the VEGA ISA level, as opposed to the intermediate language level. This page will give you a general overview of how to use this model, the software stack the model uses, and provide resources that detail the model and how it is implemented.
 
 ## **Using the model**
 
-Currently, the GCN3 GPU model in gem5 is supported on the stable and develop branch.
+Currently, the AMD GPU model in gem5 is supported on the stable and develop branch.
 
-The [gem5 repository](https://github.com/gem5/gem5) comes with a dockerfile located in `util/dockerfiles/gcn-gpu/`. This dockerfile contains the drivers and libraries needed to run the GPU model. A pre-built version of the docker image is hosted at `gcr.io/gem5-test/gcn-gpu:v22-0`.
+The [gem5 repository](https://github.com/gem5/gem5) comes with a dockerfile located in `util/dockerfiles/gcn-gpu/`. This dockerfile contains the drivers and libraries needed to run the GPU model. A pre-built version of the docker image is hosted at `ghcr.io/gem5-test/gcn-gpu:v23-1`.
```

Comment:
pretty sure "ghcr" should be "gcr"?

Reply:
I'll need to dig up the  specific patch, but this is part of the change and is in the commit message.  If I remember correctly, I believe that ghcr is free and gcr is not.  I am referring to a change @BobbyRBruce made a while back and there are many commits switching over to ghcr so I made this change while I was editing this file

Reply:
Ok, as long as you checked this, it's fine with me.